### ISSUE TITLE
problem in Finetuning Bacformer for phenotypic traits prediction

### ISSUE THREAD

**Issue by lz-pfg** [macwiatrak/Bacformer#30]
Hello authors,

Thanks for working on tutorials. When I run this traits prediction, in line 
"collate_genome_samples_fn = partial(collate_genome_samples, SPECIAL_TOKENS_DICT["PAD"], 7000, 1000)"
it reports error "collate_genome_samples() takes from 0 to 3 positional arguments but 4 were given".
Then I find the corresponding function with the arguments like this 
"def collate_genome_samples(pad_token_id: int = 0, max_n_contigs: int = 1000, samples: list[dict] = None, ) -> dict[str, torch.Tensor]:"

So I believe the pad_token_id is fixed to SPECIAL_TOKENS_DICT["PAD"] and max_n_contigs is fixed to 7000. So what is the number 1000?

To continue the prediction, I just remove the 1000 and try to train the model. However, it reports "collate_genome_samples() got multiple values for argument 'pad_token_id'. " Then I remove the SPECIAL_TOKENS_DICT["PAD"]. Later, I modify the keys into protein_embeddings and label, but it reports other errors "ArrayRef: invalid slice, N = 1; size = 0". So in sum, it cannot run. Could you help with this?

Thanks so much!
